src/storage/src/storage_failpoints/test_hummock.rs

```diff
@@ -79,10 +77,11 @@ async fn test_failpoint_state_store_read_upload() {
     local_version_manager
         .refresh_version(meta_client.as_ref())
         .await;
-
+    // clear block cache
+    sstable_store.clear_block_cache();
     fail::cfg(mem_read_err, "return").unwrap();
 
-    let result = hummock_storage.get(b"bb".as_ref(), 2).await;
+    let result = hummock_storage.get(&anchor, 2).await;
```

Comment:
Seems that anchor is `"aa"`, while the original one is `"bb"`. Is this consistent with the original code?

Reply:
I think it is consistent. This test is testing get with object store returuing an error on read. `"bb"` is an non-existent key and it is possible that bloom filter will filter it out before reaching object store so it is better to use an existent key here. cc @xxhZs

Reply:
yes , I think it is consistent too

Reply:
I suggest we explicitly disable bloom filter when failpoints is enabled, also for cache policy. `failpoints` provides a function `fail::has_failpoints` to check whether it is enabled (simply a `cfg!(feature = "failpoints"))`.